tests/hypothesis_strategies/kernels.py

```diff
@@ -35,3 +47,18 @@ def kernels(draw: st.DrawFn):
         )
     else:
         return base_kernel
+
+
+@st.composite
+def kernels(draw: st.DrawFn):
```

Comment:
can the composites be restructured to being able to be called like `kernels` and not `kernels()` ?
perhaps there are better ways but doing `def _kernels...; kernels = _kernels()` works already